src/OptionContainer.cpp

```diff
@@ -32,21 +33,18 @@ extern thread_local std::string thread_id;
 
 // IMPLEMENTATION
 
-OptionContainer::OptionContainer()
-{
+OptionContainer::OptionContainer() {
```

Comment:
Codacy found an issue: [Member variable 'OptionContainer::myDebug' is not initialized in the constructor.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238856)